DEVELOPMENT.md

```diff
@@ -69,22 +69,11 @@ We adhere to the [general Sourcegraph principles for testing](https://docs.sourc
 
 ### Patch releases
 
-If a backwards-compatible change is made _after_ a backwards-incompatible one, the backwards-compatible one should be re-released to older instances that support it.
+If a backwards-compatible change, such as a bug fix, is made _after_ a backwards-incompatible one, the backwards-compatible one should be re-released to older instances that support it.
 
 A Sourcegraph instance returns the highest patch version with the same major and minor version as `MinimumVersion` as defined in the instance. Patch versions are reserved solely for non-breaking changes and minor bug fixes. This allows us to dynamically release fixes for older versions of `src-cli` without having to update the instance.
 
-To release a bug fix or a new feature that is backwards compatible with one of the previous two minor version of Sourcegraph, cherry-pick the changes into a patch branch and re-releases with a new patch version. 
-
-For example, suppose we have the the recommended versions.
-
-| Sourcegraph version | Recommended src-cli version |
-| ------------------- | --------------------------- | 
-| `3.100`             | `3.90.5`                    |
-| `3.99`              | `3.85.7`                    |
-
-If a new feature is added to a new `3.91.6` release of src-cli and this change requires only features available in Sourcegraph `3.99`, then this feature should also be present in a new `3.85.8` release of src-cli. Because a Sourcegraph instance will automatically select the highest patch version, all non-breaking changes should increment only the patch version. 
-
-Note that if instead the recommended src-cli version for Sourcegraph `3.99` was `3.90.4` in the example above, there is no additional step required, and the new patch version of src-cli will be available to both Sourcegraph versions.
```

Comment:
This example just confused me, since 99% of the time the recommended src-cli version matches the Sourcegraph version, so I decided to just remove it. Perhaps this was more common historically.